### ISSUE TITLE
Jack2 on Ventura (13.5) - no input signal

### ISSUE THREAD

**Issue by klangschmied** [jackaudio/jack2#958]
## Describe the bug

After installing "jack2-macOS-intel-v1.9.22" on Ventura MacOS 13.5, there is no input signal whether in pd or sooperlooper, which a relying on Jack. Jack starts with QjackCtrl fine.
There was no "microphone allowing" after starting QJackctrl, sooperlooper and pd....

## Environment

* jackdmp version 1.9.22 tmpdir /tmp protocol 9
* MacOS 13.5
* jack2-macOS-intel-v1.9.22.dmg

## Steps To Reproduce

```bash
# Paste a minimal code example here (e.g. about how you started JACK)
```

### Expected vs. actual behavior

A clear and concise description of what you expected to happen vs. what happened.

**Comment by bmx666** [comment]
I installed [qjackctl from brew](https://formulae.brew.sh/formula/qjackctl) and download the latest MacOS universal package from [build](https://github.com/jackaudio/jack2/actions/workflows/build.yml) - [jack2-macOS-universal-0d730dd3](https://github.com/jackaudio/jack2/suites/17712186583/artifacts/1013960150)
Run **qjackctl** from command line, then go into Settings -> Security -> Microphone -> qjackctl -> Enable.
Then restart **qjackctl**.
Anyway I got a lot of errors during configuration different input devices from jackd. AirPods also doesn't work, jackd returns "informative" errors 😢

**Comment by klangschmied** [comment]
Hoe do I run qjackctrl from commandline? 
`open qjackctrl.app`

**Comment by bmx666** [comment]
> Hoe do I run qjackctrl from commandline? `open qjackctrl.app`

After installation with brew, open new terminal window and type `qjackctl`.
If you not familiar with brew, please check Google.

**Comment by klangschmied** [comment]
I am familiar with brew :-)
Starting `qjackctl` in a new terminal window worked, but
there is no qjackctl enabling Settings -> Security -> Microphone -> qjackctl -> Enable

**Comment by bmx666** [comment]
> but there is no qjackctl enabling Settings -> Security -> Microphone -> qjackctl -> Enable

Did you try select buildin mic on Mac and start jackd server?

**Comment by klangschmied** [comment]
Hi bmx666, thanks a lot for your hints!
I got it to work with a Apple Headset on an Mac Mini, jackd ist enabled in the microphone securities.
BTW stopping qjackctl does not work...spinning wheel... I have to force quit it, hm? Starting again, there is no GUI, only after restarting the Mac.

**Comment by bmx666** [comment]
Yes, jackd has problems on Apple ARM64 with OS 13 and 14. It's not a qjackctl issue.

**Comment by klangschmied** [comment]
BTW, my Mac Mini 2018 is not an ARM64...is a Intel-version

**Comment by klangschmied** [comment]
Argh, stop `qjackctl`gives me a spinning wheel, only
`killall -9 jackd`
stops qjackctl
 Now I am not able any more to connect via qjackctl. There is an error in qjackctl:
`Cannot connect to server socket err = Connection refused`
Now I am lost :-(
